src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -137,8 +109,7 @@ const CallTable = ({ namespace, name, submissionId, workflowId, callName, callOb
         ]
       })
     ]),
-    failuresModalParams && h(FailuresModal, { ...failuresModalParams, callFqn: callName, onDismiss: () => setFailuresModalParams(undefined) }),
-    wizardSelection && h(CallCacheWizard, { onDismiss: () => setWizardSelection(undefined), namespace, name, submissionId, workflowId, ...wizardSelection })
```

Comment:
Presumably there is a `CallCacheWizard` element somewhere that would also benefit from removal?